packages/ez_search/ez_search.0.2.0/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+synopsis: "The ez_search project"
+description: """\
+This is the description
+of the ez_search OCaml project
+"""
```

Comment:
could you give a proper description and synopsis?

Reply:
Will do